README.md

```diff
@@ -57,21 +61,25 @@ Given the following releases of a dependency:
 - `0.1.0`
 - `1.0.0`
 - `2.0.0`
+- `2.0.5`
+- `2.0.6`
 - `2.1.0`
-
-constraints would resolve as follows:
-
-| Constraint                       | Resolution |
-| -------------------------------- | ---------- |
-| `=1.0.0`                         | `1.0.0`    |
-| `>1.0.0`                         | `2.1.0`    |
-| `>=1.0.0`                        | `2.1.0`    |
-| `<2.0.0`                         | `1.0.0`    |
-| `<=2.0.0`                        | `2.0.0`    |
-| `!=1.0.0`                        | `2.1.0`    |
-| `>1.0.0 && <2.1.0`               | `2.0.0`    |
-| `<1.0.0 \|\| >2.0.0`             | `2.1.0`    |
-| `(>0.1.0 && <2.0.0) \|\| >2.1.0` | `1.0.0`    |
+- `3.0.0`
+
+constraints conditions would match as follows:
+
+| The following condition...       | will match with versions...                          |
+| -------------------------------- | ---------------------------------------------------- |
+| `=1.0.0`                         | `1.0.0`                                              |
+| `>1.0.0`                         | `2.0.0`, `2.0.5`, `2.0.6`, `2.1.0`, `3.0.0`          |
+| `>=1.0.0`                        | `1.0.0`, `2.0.0`, `2.0.5`, `2.0.6`, `2.1.0`, `3.0.0` |
+| `<2.0.0`                         | `0.1.0`, `1.0.0`                                     |
+| `<=2.0.0`                        | `0.1.0`, `1.0.0`, `2.0.0`                            |
+| `!=1.0.0`                        | `0.1.0`, `2.0.0`, `2.0.5`, `2.0.6`, `2.1.0`, `3.0.0` |
+| `>1.0.0 && <2.1.0`               | `2.0.0`, `2.0.5`, `2.0.6`                            |
+| `<1.0.0 \|\| >2.0.0`             | `0.1.0`, `2.0.5`, `2.0.6`, `2.1.0`, `3.0.0`          |
+| `(>0.1.0 && <2.0.0) \|\| >2.0.5` | `1.0.0`, `2.0.6`, `2.1.0`,`3.0.0`                    |
+| `^2.0.5`                         | `2.0.5`, `2.0.6`, `2.1.0`                            |
```

Comment:
```suggestion
| `^2.0.5`                         | `2.0.5`, `2.0.6`, `2.1.0`                            |
| `^0.1.0`                         | `0.1.0`, `0.1.1`                                     |
```

I think a demonstration of this is worth adding as the "allows changes that do not modify the left-most non-zero element" is not obvious to those who aren't already familiar with equivalent constraint systems.